### PR TITLE
Improved examples to be easier.

### PR DESCRIPTION
**Pull request by TheGiga**
## Summary

Let's be real, no one, who needs those examples, will use subclassing style. Most of other examples are using this ^ style to show stuff.

## Information

- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...).

## Checklist

- [ ] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.

### CODE REVIEW

**Comment by UP929312** [comment]
I actually agree, if you're looking for help, you're probably using this (changed) method, I think it makes the example much easier to follow for new people, and overall clears up this example

File: examples/new_member.py [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 
+intents = discord.Intents.default()
+intents.members = True  # This intent requires "Server Member Intent" to be enabled at https://discord.com/developers
+# ^ This may give you `read-only` warning, just ignore it.
 
-class MyClient(discord.Client):
-    async def on_ready(self):
-        print(f"Logged in as {self.user} (ID: {self.user.id})")
-        print("------")
+bot = discord.Bot(intents=intents)
```
Comment (EmreTech) [inline examples/new_member.py:7]: Does this really have to be `discord.Bot`?

File: examples/new_member.py [hunk 2]
```diff
@@ -1,22 +1,22 @@
-# This example requires the 'members' privileged intent to use the Member converter.
-
 import discord
 
+intents = discord.Intents.default()
+intents.members = True  # This intent requires "Server Member Intent" to be enabled at https://discord.com/developers
```
Comment (EmreTech) [inline examples/new_member.py:4]: This isn't exactly the full link to the Developers Dashboard

**Review by Dorukyum — APPROVED** [review]
Looks good, thanks

File: examples/new_member.py (hunk 2 above)
Comment (Dorukyum) [reply examples/new_member.py:4]: It redirects there

Comment (EmreTech) [reply examples/new_member.py:4]: Nevermind about this one, I think my brain isn't working

File: examples/new_member.py (hunk 1 above)
Comment (Dorukyum) [reply examples/new_member.py:7]: I guess, since discord.Bot is a lot more used than Client, it's better to show new members to start with it.

Comment (EmreTech) [reply examples/new_member.py:7]: Good point

**Pull request merged**